parm/input.mom6.nml.IN

```diff
@@ -158,8 +158,8 @@
        random_clds    = .true.
        trans_trac     = .false.
        cnvcld         = .true.
-       imfshalcnv     = @[IMFSHALCNV]
-       imfdeepcnv     = @[IMFDEEPCNV]
+       imfshalcnv     = 2
```

Comment:
I think we probably want to keep imshalcnv and imfdeepcnv as settable parameters in the input.mom6.nml.IN. 
In the benchmark.nml.IN they should remain fixed values.

Reply:
Reverted.  2 is the default value in fv3.input so this should not effect any answers.